client/js/Ui/core.js

```diff
@@ -32,6 +32,11 @@ SCWeb.ui.Core = {
               self.resolveElementsAddr('body'),
             ]).then(function () {
 
+                $('#header').removeClass('no_display');
```

Comment:
You can concatenate like `$('#header, #search-panel, #footer').removeClass('no_display');`